packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -181,4 +181,122 @@ describe('<DataGrid />', () => {
       );
     });
   });
+
+  describe('column width', () => {
+    before(function beforeHook() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        // Need layouting
+        this.skip();
+      }
+    });
```

Comment:
I think that it would be simpler to move the describe under the `layout` section, it already includes this clause.

Reply:
done